src/Xamarin.Android.Build.Tasks/Tasks/GeneratePackageManagerJava.cs

```diff
@@ -216,6 +219,19 @@ void AddEnvironment ()
 						continue;
 					}
 
+					if (lineToWrite.StartsWith ("System.Globalization.Invariant=", StringComparison.Ordinal)) {
```

Comment:
These should use the "real" environment variable names: `DOTNET_SYSTEM_GLOBALIZATION_INVARIANT`, `DOTNET_SYSTEM_GLOBALIZATION_USENLS`, `DOTNET_SYSTEM_GLOBALIZATION_APPLOCALICU`:

https://github.com/dotnet/runtime/blob/ce63fb0a4145891760f9530674e93f237dc627c5/src/mono/mono/mini/monovm.c#L185-L194

Rationale: `System.Globalization.Invariant` can't be a Unix environment variable, so possible one-off testing may be hindered (and is inconsistent).